src/compose/ComposeBox.js

```diff
@@ -184,8 +190,14 @@ class ComposeBox extends PureComponent<Props, State> {
     dispatch(draftUpdate(narrow, message));
   };
 
-  handleMessageAutocomplete = (message: string) => {
+  handleMessageAutocomplete = (message: string, completion: string, completionType: string) => {
```

Comment:
>If that's the case, let's see if we can say that explicitly with the types, so instead of string, it would be ':' | '#' | '@'. A JSDoc would certainly help with this as well, as I've mentioned in another comment. slightly_smiling_face

I had tried `':' | '#' | '@'` earlier as the type, but it causes the Flow error, because the value is derived from some plumbing in `getAutocompleteFilter`, and Flow is not intelligent enough to figure out that only these three are possible values, instead of a complete string.